df.units.xml

```diff
@@ -180,23 +180,71 @@
         <enum-item name='PEACE'/>
         <enum-item name='KNOWLEDGE'/>
     </enum-type>
-
+  
     <enum-type type-name='goal_type' base-type='int32_t'>
-        <enum-item name='STAY_ALIVE'/>
-        <enum-item name='MAINTAIN_ENTITY_STATUS'/>
-        <enum-item name='START_A_FAMILY'/>
-        <enum-item name='RULE_THE_WORLD'/>
-        <enum-item name='CREATE_A_GREAT_WORK_OF_ART'/>
-        <enum-item name='CRAFT_A_MASTERWORK'/>
-        <enum-item name='BRING_PEACE_TO_THE_WORLD'/>
-        <enum-item name='BECOME_A_LEGENDARY_WARRIOR'/>
-        <enum-item name='MASTER_A_SKILL'/>
-        <enum-item name='FALL_IN_LOVE'/>
-        <enum-item name='SEE_THE_GREAT_NATURAL_SITES'/>
-        <enum-item name='IMMORTALITY'/>
-        <enum-item name='MAKE_A_GREAT_DISCOVERY'/>
-        <enum-item name='ATTAINING_RANK_IN_SOCIETY' since='v0.47.01'/>
```

Comment:
I just checked against a disassembly, and it turns out that these two enum names were wrong - they're supposed to be "ATTAIN_RANK_IN_SOCIETY" and "BATHE_WORLD_IN_CHAOS" to match the goal tokens that can be specified in the raws.

As long as you're adding the new attributes, it wouldn't hurt to fix these too.

Reply:
I'll take care of this after I merge.